mmv1/third_party/terraform/tests/resource_container_node_pool_test.go.erb

```diff
@@ -2274,6 +2274,10 @@ resource "google_container_node_pool" "np_with_gpu" {
       type  = "nvidia-tesla-a100"
       gpu_partition_size = "1g.5gb"
       count = 1
+	  gpu_sharing_config {
+		gpu_sharing_strategy = "TIME_SHARING"
+		max_shared_clients_per_gpu = 2
+	  }
```

Comment:
spacing appears out of place. Blocks declaration could be in inline with the rest of the variable declarations.

Reply:
done